docs/reference-docs/app/scaling.md

```diff
@@ -191,8 +197,8 @@ class OrchestratorWorker(Celery):
         self.process_broadcast_fn = process_broadcast_fn
 
```

Comment:
While we're updating this; in our OrchestratorWorker the on_init starts off like this:

```
    def on_init(self) -> None:
        start_debugger()
```

Which is imported with `from nwastdlib.debugging import start_debugger`. This allows for interactive debugging.